generator/sbpg/targets/resources/rust/test/sbp_tests_template.rs

```diff
@@ -51,7 +51,7 @@ fn test_(((s.suite_name|snake_case)))()
         match &sbp_msg {
             sbp::messages::Sbp::(((t.msg.name|lower_acronyms)))(msg) => {
                 let msg_type = msg.message_type().unwrap();
-                assert_eq!( msg_type, (((t.msg_type))), "Incorrect message type, expected (((t.msg_type))), is {}", msg_type);
+                assert_eq!( msg_type, (((t.msg_type))), "Incorrect message type, expected (((t.msg_type))), is {msg_type}");
```

Comment:
msg_type moved in between the curly braces to fix all lints in automatically generated tests